### PR TITLE
add scheduler multi profile test

### PR DESCRIPTION
**Pull request by JaneLiuL**
#### What type of PR is this?
/kind  failing-test
/sig scheduling

#### What this PR does / why we need it:
The scheduler define `multiProfilesConfig` but do not test in the code.
#### Which issue(s) this PR fixes:

Fixes # Add  `multiProfilesConfig` test

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: cmd/kube-scheduler/app/server_test.go [hunk 1]
```diff
@@ -230,6 +231,104 @@ profiles:
...
+			wantPlugins: map[string]*config.Plugins{
+				"profile-default-plugins": {
+					QueueSort: config.PluginSet{
+						Enabled: []config.Plugin{
+							{Name: names.PrioritySort},
+						},
+					},
+					PreFilter: config.PluginSet{
+						Enabled: []config.Plugin{
+							{Name: names.NodeResourcesFit},
```
Comment (alculquicondor) [inline cmd/kube-scheduler/app/server_test.go:249]: just use literals as the rest of the tests in this file.

File: cmd/kube-scheduler/app/server_test.go [hunk 2]
```diff
@@ -230,6 +231,104 @@ profiles:
 				},
 			},
 		},
+		{
+			name: "multi profiles files",
```
Comment (alculquicondor) [inline cmd/kube-scheduler/app/server_test.go:235]: Actually, we don't need this test **and** the one with one profile. Let's just leave the multiple profiles one.

Comment (alculquicondor) [reply cmd/kube-scheduler/app/server_test.go:235]: In other words, remove the test case named `component configuration`

**Review by damemi — COMMENTED** [review]
@JaneLiuL you also need something in the release-note block, you can just put `NONE`

File: cmd/kube-scheduler/app/server_test.go [hunk 3]
```diff
@@ -166,46 +166,6 @@ profiles:
 				"default-scheduler": defaults.PluginsV1beta2,
 			},
 		},
-		{
-			name: "component configuration",
-			flags: []string{
-				"--config", pluginConfigFile,
```
Comment (alculquicondor) [inline cmd/kube-scheduler/app/server_test.go:172]: can you remove the `pluginConfigFile` variable?

**Pull request closed without merging**